pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -380,15 +423,40 @@ class CreateVM extends React.Component {
 
                 <hr />
 
-                <MemorySelectRow label={_("Memory")}
-                                 id={"memory-size"}
-                                 value={this.state.memorySize}
-                                 initialUnit={this.state.memorySizeUnit}
-                                 onValueChange={e => this.onChangedEventValue('memorySize', e)}
-                                 onUnitChange={e => this.onChangedValue('memorySizeUnit', e)} />
+                <label className="control-label" htmlFor="storage-pool-select">
+                    {_("Storage")}
+                </label>
+                <Select.Select id="storage-pool-select"
+                               initial={this.state.storagePool}
+                               onChange={e => this.onChangedValue('storagePool', e)}
+                               enabled={this.state.sourceType != EXISTING_DISK_IMAGE_SOURCE}>
```

Comment:
This is part of the reason why I shuffled everything around in the mockups — so we can have source & destination next to each other so they have a bit more context when things go away.

While some of the design suggestions should probably still be in another PR (such as adjusting label text, changing the system connection from a dropdown to radio buttons), changing the order probably should be in this PR so storage makes sense when this PR lands.

Reply:
So I moved destination next to the source.
Also now i somebody chooses source: existing disk image, then all the destination option disappear instead of just being disabled.
![Screenshot from 2019-04-24 10-57-03](https://user-images.githubusercontent.com/42733240/56646439-db70e700-667f-11e9-8ace-591df085e062.png)
![Screenshot from 2019-04-24 10-57-12](https://user-images.githubusercontent.com/42733240/56646446-ddd34100-667f-11e9-9433-402b3206a0fe.png)

Reply:
Much better! Thanks!

Looking forward to the other items getting resorted too. (In the follow-up PR.)